README.md

```diff
@@ -1,2 +1,113 @@
-# js-ipfs-repo
-Implementation of the IPFS repo spec (https://github.com/ipfs/specs/tree/master/repo) in JavaScript
+js-ipfs-repo
+============
+
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io) [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/) [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+
+> Implementation of the IPFS repo spec (https://github.com/ipfs/specs/tree/master/repo) in JavaScript
+
+## API
+
+### `Repo`
+
+Constructor, accepts a path and options:
+
+```js
+var Repo = require('js-ipfs-repo')
+var repo = new Repo('/Users/someone/.ipfs', {adaptor: 'fs'})
+```
+
+Options:
+
+  - `adaptor`: String with the adaptor. Defaults to `fs`
+
+### `#version`
+
+Read/Write the version number of that repository.
+
+```js
+repo.version().read(function (err, num) {
```

Comment:
I'm "injecting" the store on each of those (https://github.com/ipfs/js-ipfs-repo/pull/2/files#diff-1fdf421c05c1140f6d71444ea2b27638R36). I can improve that I guess.